packages/react/src/utils/profile.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  ICompletionScores,
+  IUser,
+  IUserDetails,
+  IWeb3mailPreferences,
+} from "../types";
+
+const pointsByUserFields = {
```

Comment:
@marsian83 @pranav-singhal that part is specific to starterkit, don't copy it here